test/playwright/tests/exploreAoi.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { test, expect } from '../pages/basePage';
+
+test.describe('Area of Interest (AOI) Analysis', () => {
+  test.beforeEach(
+    async ({ page, explorePage, consentBanner, datasetSelectorComponent }) => {
+      let pageErrorCalled = false;
+      // Log all uncaught errors to the terminal to be visible in trace
+      page.on('pageerror', (exception) => {
+        console.log(`Uncaught exception: "${JSON.stringify(exception)}"`);
+        pageErrorCalled = true;
+      });
+
+      const mapboxResponsePromise = page.waitForResponse(
+        /api\.mapbox.com\/v4\/mapbox\.mapbox-streets-v8/i
+      );
+
+      // Given that I am on the map view (explore page)
```

Comment:
if we want the gherkin steps to show up in the test report, we can wrap these in the playwright [test.step](https://playwright.dev/docs/api/class-test#test-step)

```
test('Given that I am on the map view (explore page)', async ({ page }) => {
    await page.goto('/exploration');
});
and that text appears in the saved reports